NitroxServer/Program.cs

```diff
@@ -34,7 +34,7 @@ static void Main(string[] args)
             ConsoleCommandProcessor CmdProcessor = NitroxServiceLocator.LocateService<ConsoleCommandProcessor>();
             while (server.IsRunning)
             {
-                CmdProcessor.ProcessCommand(Console.ReadLine());
+                //CmdProcessor.ProcessCommand(Console.ReadLine()); I dont think is is required ...
```

Comment:
Why isn't this required? How would the server console handle commands?

Reply:
I need to look further into it , i just know they still work without it...